app/controllers/api/compact_index_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::V2::CompactIndexController < Api::BaseController
+class Api::CompactIndexController < Api::BaseController
```

Comment:
If I understatood @dwradcliffe correctly, this should be only `CompactIndexController` instead of `Api::CompactIndexController`. Thanks for the patch!